.github/workflows/build.yml

```diff
@@ -70,8 +70,6 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - macOS-10.15
-          - windows-2019
     runs-on: ${{ matrix.os }}
```

Comment:
WDYT about using the [localstack](https://hub.docker.com/r/localstack/localstack) image with the Github Action [syntax](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer)  for using the Docker container directly?

Reply:
then it won't allow to use it for local dev/tests if I get this right, also, does it support custom dockerfile?

Reply:
@fvaleye I've looked more into that and yes, that would work for us, but still we won't be able to boot up s3 locally then

Reply:
@mosyp thanks for the test!
I get your point, we would have to launch `localstack` into two distinct ways, one with the CI and one for our local tests. We can keep the custom Docker image but it would be good to make it work for the CI.

Reply:
@fvaleye please check out my today's changes, all of the comments are addressed and resolved, so there's no more custom images etc. But I still cannot use only localstack for the setup of data since then it won't be possible to wait for its launch/init outside of the container

Reply:
Well a little update, we're able now to use a single localstack image for both localstack itself and another for setup only

Reply:
Got it!